gtda/mapper/utils/visualization.py

```diff
@@ -138,8 +138,11 @@ def _get_node_colors(data, is_data_dataframe, node_elements,
             node_elements, color_data, summary_stat=node_color_statistic)
 
     # Normalise node colours in range [0,1] for colorscale mapping
-    node_colors = (node_colors - np.min(node_colors)) / \
-        (np.max(node_colors) - np.min(node_colors))
+    if any(np.logical_not(np.isfinite(node_colors))):
```

Comment:
should we add a comment here? something like "Check if node_colors contains NaNs" 

although i generally dislike code comments, i think in this case it could help since the numpy functions require some mental parsing i'd prefer to avoid future versions of ourselves wondering about 😄